apps/user-portal/src/components/federated-associations/federated-associations.tsx

```diff
@@ -105,10 +109,51 @@ export const FederatedAssociations = (props: FederatedAssociationsPropsInterface
             });
     };
 
+    /**
+     * Pops up a model requesting confirmation before deleting 
+     */
+    const deleteConfirmation = (): React.ReactElement => {
+        return (
+            <Modal
+                dimmer="blurring"
+                size="mini"
+                open={ confirmDelete }
+                onClose={ () => { setConfirmDelete(false); } }
+            >
+                <Modal.Content>
+                    { t("views:components.federatedAssociations.deleteConfirmation") }
+                </Modal.Content>
+                <Modal.Actions>
+                    <Button
+                        className="link-button"
+                        onClick={ () => {
+                            setId(null);
+                            setConfirmDelete(false);
+                        } }
+                    >
+                        { t("common:cancel") }
+                    </Button>
+                    <Button
+                        primary
+                        onClick={ () => {
+                            removeFederatedAssociation(
+                                id
+                            );
```

Comment:
Make it single line

Reply:
Shall we fix this if it hasn't been fixed yet @thivi